pom.xml

```diff
@@ -280,6 +287,18 @@
 							<version>2.29</version>
 						</driver>
 						<driver>
+							<name>chromedriver</name>
+							<platform>linux</platform>
+							<bit>32</bit>
+							<version>2.29</version>
```

Comment:
Please remove the linux web drivers from the project. These makes the 'standalone.zip' really huge and people running Linux tend to be technical enough to add the drivers themselves...